projects/fsastorefrontlib/src/core/checkout/store/reducers/address-verification.reducer.ts

```diff
@@ -1,41 +0,0 @@
-import { CheckoutActions, AddressVerificationState } from '@spartacus/checkout/core';
```

Comment:
We had the same verification reducer as Spartacus team. probably we had to copy-paste it bcs it wasn't exported or something. As they removed it, probably it is not needed anymore on our side either